provider/aws/formation/rack.json

```diff
@@ -68,29 +127,29 @@
+      "ap-southeast-2": { "Ami": "ami-012bb964803d474e5", "ArmAmi": "ami-0f2d4206602a56c27", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "783225319266", "Fargate": "Yes" },
+      "ca-central-1":   { "Ami": "ami-0941553a556a284dc", "ArmAmi": "ami-01866bd198d129564", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "985666609251", "Fargate": "No"  },
+      "eu-central-1":   { "Ami": "ami-09219f0735c86449f", "ArmAmi": "ami-0becc22cd1c303541", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "054676820928", "Fargate": "Yes" },
+      "eu-north-1":     { "Ami": "ami-0898735051d6a03e2", "ArmAmi": "ami-0356e81e07ac1aa76", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "897822967062", "Fargate": "Yes" },
+      "eu-south-1":     { "Ami": "ami-0287efd4023ea25ff", "ArmAmi": "ami-0a1c91dce96b88473", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "635631232127", "Fargate": "Yes" },
+      "eu-west-1":      { "Ami": "ami-06c11ea68c61e5570", "ArmAmi": "ami-0c2511f2e0582024b", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "156460612806", "Fargate": "Yes" },
+      "eu-west-2":      { "Ami": "ami-0a078ce6abe5fcb1b", "ArmAmi": "ami-0c6fa7cace3a84a70", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "652711504416", "Fargate": "Yes" },
+      "eu-west-3":      { "Ami": "ami-06f2ca755fea150d5", "ArmAmi": "ami-0d3b012ab73f94a25", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "009996457667", "Fargate": "Yes" },
+      "me-south-1":     { "Ami": "ami-0a80f099ac41496b7", "ArmAmi": "ami-0cd2c2eef8ea85565", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "076674570225", "Fargate": "Yes" },
+      "sa-east-1":      { "Ami": "ami-08a6250517f5dd2be", "ArmAmi": "ami-04215bcb38f33ad27", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "507241528517", "Fargate": "Yes" },
+      "us-east-1":      { "Ami": "ami-0e43fd2a4ef14f476", "ArmAmi": "ami-0bfc02e7ac93240fd", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "127311923021", "Fargate": "Yes" },
+      "us-east-2":      { "Ami": "ami-023ef2526f4bf79c9", "ArmAmi": "ami-0288951ce92175bdb", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "033677994240", "Fargate": "Yes" },
+      "us-west-1":      { "Ami": "ami-0e92f5873da29c633", "ArmAmi": "ami-00e41e0af4393d3cb", "EFS": "Yes", "ThirdAvailabilityZone": "No",  "ELBAccountId": "027434742980", "Fargate": "No"  },
+      "us-west-2":      { "Ami": "ami-05a7414872af4b564", "ArmAmi": "ami-09ab5d76b117b41b0", "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "797873946194", "Fargate": "Yes" },
+      "us-gov-east-1":  { "Ami": "ami-04f633b669ee7b61c", "ArmAmi": "",                      "EFS": "Yes", "ThirdAvailabilityZone": "Yes", "ELBAccountId": "190560391635", "Fargate": "Yes" },
```

Comment:
We have to manually go into the us-gov regions to get the AMIs but that's very, very low priority